plugins/woocommerce/includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -1401,13 +1401,13 @@ protected function get_wizard_available_in_cart_payment_gateways() {
 		) . '</p>';
 
 		return array(
-			'stripe'          => array(
-				'name'        => __( 'WooCommerce Stripe Gateway', 'woocommerce' ),
-				'image'       => WC()->plugin_url() . '/assets/images/stripe.png',
-				'description' => $stripe_description,
-				'class'       => 'checked stripe-logo',
-				'repo-slug'   => 'woocommerce-gateway-stripe',
-				'settings'    => array(
+			'stripe'               => array(
+				'name'             => __( 'WooCommerce Stripe Gateway', 'woocommerce' ),
+				'image'            => WC()->plugin_url() . '/assets/images/stripe.png',
+				'description'      => $stripe_description,
+				'class'            => 'checked stripe-logo',
+				'repo-slug'        => 'woocommerce-gateway-stripe',
+				'settings'         => array(
```

Comment:
Here but also in one or more of the following blocks, can we verify the spacing before the `=>` is correct? PHPCS is complaining and I think the problem is alignment should be "per level". That is, within the inner array we don't need alignment with the arrows from the outer array.

```suggestion
				'name'        => __( 'WooCommerce Stripe Gateway', 'woocommerce' ),
				'image'       => WC()->plugin_url() . '/assets/images/stripe.png',
				'description' => $stripe_description,
				'class'       => 'checked stripe-logo',
				'repo-slug'   => 'woocommerce-gateway-stripe',
				'settings'    => array(
```

Etc, repeating as needed.